### DISCUSSION TITLE
[BUG] Get-PnPSite: The remote server returned an error: (401) Unauthorized.

### DISCUSSION THREAD

**Discussion by yesh-rajawat-robomq** [pnp/powershell#3390] [General]
### Expected behavior 
I should be able to perform all the commands without getting Unauthorized error
This error is not limited to the command Get-PnPSite

### Actual behavior 
![image](https://github.com/pnp/powershell/assets/122252877/74e0b47b-4f8a-4808-990f-ebd7ad8aca4b)

### Steps to reproduce behavior

1. Use the following reference to create azure ad app 
[https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azuread](url)
2. Instead of using certificate, use clientId and client secret.
`Connect-PnPOnline -Url "tenantName-admin.sharepoint.com" -ClientId $clientId -ClientSecret $clientSecret`
3. `Get-PnPSite`
### What is the version of the Cmdlet module you are running?
 2.2.0
### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows
- [. ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by SayCloud** [comment]
I encounter same issue. I'm running the script on Windows server 2016 and PowerShell 7.3.6
